Sluggo/View Controllers/AdminTableViewController.swift

```diff
@@ -23,7 +23,17 @@ class AdminTableViewController: UITableViewController {
 
     @IBSegueAction func createUsersList(_ coder: NSCoder) -> MemberListViewController? {
         let memberVC = MemberListViewController(coder: coder, identity: identity)
-        memberVC?.generateSegueableController = { _ in
+        memberVC?.generateSegueableController = { oldIdentity, record in
+            if oldIdentity.authenticatedUser?.pk == record?.owner.id {
```

Comment:
oldIdentity seems to imply that the identity is outdated, which should never occur given identity is stateful and passed along. I don't think there's anything particularly mutating the identity here, unless I'm missing something - is this just referring to the fact that the identity comes from the member view controller? In that case I think it would be better to refer to it by some other name.